launch/multiple_cams.launch

```diff
@@ -19,9 +19,9 @@
         <param name="frame_id"        type="string" value="0"        />
         <param name="num_cams_in_bus" type="int"    value="2"        />
         <param name="bw_safetyratio"  type="double" value="1.0"      />
-        <param name="publish_xi_image_info" type="bool" value="false"/>
+        <param name="publish_xi_image_info" type="bool" value="true"/>
```

Comment:
I feel like we need a calibration file if we are going to set this to true

Reply:
this is actually the extra ximea camera info that is also embedded in the image. See line 767 in the ximea_ros_cam.cpp for all of the outputs, or https://www.ximea.com/support/wiki/apis/XiAPI_Manual#xiGetImage for the entire header structure